.github/workflows/e2e.yaml

```diff
@@ -80,6 +80,12 @@ jobs:
         run: |
           kubectl -n impersonation apply -f ./config/testdata/impersonation
           kubectl -n impersonation wait kustomizations/podinfo --for=condition=ready --timeout=4m
+      - name: Run image overide tests
+        run: |
+          kubectl create ns override-test
+          kubectl -n override-test apply -f ./config/testdata/overrides
+          ACTUAL_TAG=$(kubectl -n override-test get deployments podinfo -o jsonpath='{.spec.template.spec.containers[0].image}' | cut -f2 -d ":")
```

Comment:
Before this check, we need to wait for the kustomization to reconcile first with:

```
kubectl -n override-test wait kustomizations/podinfo --for=condition=ready --timeout=1m
```